libs/utils/android/workloads/gmaps.py

```diff
@@ -90,20 +91,34 @@ def run(self, out_dir, location_search, swipe_count=5, collect=''):
         # Allow the activity to start
         sleep(1)
 
-        self.tracingStart()
+        dummy = DummyTarget(self._target)
```

Comment:
If the list of actions is directly grabbed from the accumulation list, maybe there should be a setter property for it ? A user might want to filter out some commands with regexps or something.